.ci/smoke-tests.groovy

```diff
@@ -78,6 +79,11 @@ pipeline {
       }
     }
   }
+  post {
+    failure {
+      notifyBuildResult(slackComment: true, slackNotify: true)
+    }
```

Comment:
I'd say to run this always similar pattern done in https://github.com/elastic/apm-pipeline-library/blob/467418b3dc74c766080235ca02f57a8bac7b22ab/resources/JenkinsfileTemplate.groovy#L392-L396

> * slackNotify: Whether to send or not the slack notifications, by default it sends notifications on Failed builds that are not pull request.


https://github.com/elastic/apm-pipeline-library/blob/467418b3dc74c766080235ca02f57a8bac7b22ab/vars/notifyBuildResult.groovy#L67



```suggestion
    cleanup {
      notifyBuildResult(slackComment: true)
    }
```

If needed you can add some headers to the slack comment , see https://github.com/elastic/apm-pipeline-library/blob/main/vars/notifyBuildResult.txt#L33

Reply:
Do we want to avoid running report failure on PR?

Reply:
IIUC, the smoke-tests pipeline will only run in branches as per https://github.com/elastic/apm-server/pull/8492#discussion_r907419202 

Did it change that?

Reply:
We should be able to run smoketests on PRs, main branch, 7.17 branch, and 8.x branches.
Without the argument `slackNotify`, we will drop the report failure notification on PRs.

Reply:
Can you confirm if that's the case for PRs? I cannot see any references in  https://github.com/elastic/apm-server/issues/8303

IMO, I'd say no notifications on PRs, the GitHub checks and GitHub build comments should be the ones driving those notifications on a PR basis, without any lack notification.

Reply:
We can run smoketests on PRs only on-demand AFAIK (https://github.com/elastic/apm-server/blob/main/.ci/jobs/apm-server-smoketests-mbp.yml#L16).
I don't think it's a requirement but more "a maybe nice to have".

Reply:
For the time being, I'd say no notifications on a PR basis as we might need to revisit the user experience in terms of running the smoke tests. For some other cases, we provide some integrations with GitHub commands or GitHub tags, so anyone at Elastic, could trigger a particular stage/build without the need to use the Jenkins UI

Reply:
In general the smoke tests should be run on a daily basis, not per PR. It would be nice (but can be added later if it doesn't fit this scope) to be able to trigger them on a PR basis with github commands, similar as to what we can use for other tests on PRs already (e.g. `/test` command).